src/calculateRentalCost.js

```diff
@@ -3,8 +3,25 @@
  *
  * @return {number}
  */
+
+const DAY_RENT = 40;
+const LONG_TERM = 7;
+const SHORT_TERM = 3;
+const LONG_TERM_DISCOUNT = -50;
+const SHORT_TERM_DISCOUNT = -20;
+
 function calculateRentalCost(days) {
-  // write code here
+  const TOTAL_RENT = DAY_RENT * days;
```

Comment:
```suggestion
  const currentPrice = DAY_RENT * days;
```
Variable names should be in camelCase